vocabularies/Common.xml

```diff
@@ -707,6 +707,16 @@ The minimum structure is
 
       <Term Name="ValueListWithFixedValues" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Property Parameter">
         <Annotation Term="Core.Description" String="If specified as true, there's only one value list mapping and its value list consists of a small number of fixed values" />
+        <Annotation Term="Validation.ApplicableTerms">
+          <Collection>
+            <String>ValueListShowValues"</String>
+          </Collection>
+        </Annotation>
+      </Term>
+
+      <Term Name="ValueListShowValues" Type="Core.Tag" Nullable="false" DefaultValue="true" AppliesTo="Annotation">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="A value list with a very small number of fixed values, can decide to show all values immediately" />
       </Term>
```

Comment:
Is it planned to apply this term to other annotations later? Otherwise, I would prefer a new term `Common.ValueListWithImmediateValues` as an alternative to `Common.ValueListWithFixedValues`.

Reply:
The reason I did it that way was that the value list should have fixed (means a small number of) values. Only for these value lists I would allow it.

Reply:
Thank you for the proposed change!